src/JNI/include/com/amazonaws/kinesis/video/producer/jni/KinesisVideoClientWrapper.h

```diff
@@ -54,8 +54,8 @@
 class KinesisVideoClientWrapper
 {
     CLIENT_HANDLE mClientHandle;
-    JavaVM *mJvm;
-    jobject mGlobalJniObjRef;
+    static JavaVM *mJvm;
```

Comment:
should we add a comment why we need to revise this scope?

Reply:
Addressed in the most recent commit